src/components/dashboard/main-content/course-enrollments/course-cards/styles/_CourseCard.scss

```diff
@@ -44,4 +45,14 @@
       display: block;
     }
   }
+
+  .custom-course-card-badge {
+    line-height: 1.75;
+    width: fit-content;
+    height: 30px;
+  }
+
+  .assigned-course-card-badge {
+    background: #03C7E8;
```

Comment:
Generally, hardcoded hex values prevent theming of the application as it doesn't rely on the design system and makes the colors more difficult to maintain long term. The badge itself should already have this background color when running the `@edx/brand-edx.org` theme as well.